main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
```

Comment:
The variable name loop is confusing, try using counter to make this code more readable.